nmostesting/suites/IS0401Test.py

```diff
@@ -1296,6 +1296,50 @@ def test_20(self, test):
 
         return test.PASS()
 
+    def test_20_01(self, test):
+        """Sender manifests use the expected Content-Type"""
+
+        valid, response = self.do_request("GET", self.node_url + "senders")
+        if not valid or response.status_code != 200:
+            return test.FAIL("Unexpected response from the Node API: {}".format(response))
+        try:
+            access_error = False
+            content_type_warn = None
+            node_senders = response.json()
+            for sender in node_senders:
+                if not sender["transport"].startswith("urn:x-nmos:transport:rtp"):
+                    continue
+
+                href = sender["manifest_href"]
+                if not href:

```

Comment:
🤔 is there an unhandled KeyError? Or a hidden schema check?

Reply:
An auto test should have FAILed, but that doesn't prevent this test running and raising an uncaught exception. Next commit adds a catch here - and another 18 places there wasn't one in this test suite...